libs/SmartStore/SmartStore/Classes/SFSmartStore.m

```diff
@@ -827,41 +827,82 @@ - (BOOL)createExternalStorageDirectory:(NSString *)soupTableName {
 - (BOOL)saveSoupEntryExternally:(NSDictionary *)soupEntry
                     soupEntryId:(NSNumber *)soupEntryId
                   soupTableName:(NSString *)soupTableName {
+    
+    // Helper to log debug messages
+    void (^logDebug)(NSString* step) = ^(NSString* step) {
+        NSString *debugMessage = [NSString stringWithFormat:@"%@ (%@): soupEntryId: %@, soupTableName: %@",
+                                  NSStringFromSelector(_cmd),
+                                  step,
+                                  soupEntryId,
+                                  soupTableName];
+        [SFSDKSmartStoreLogger d:[self class] format:debugMessage];
```

Comment:
do we want this to be info, so we can get this info in release builds as well? rather than just debug